src/Microsoft.DocAsCode.Build.Common/Coordinate.cs

```diff
@@ -36,15 +35,9 @@ public static Coordinate GetCoordinate(string content)
                 return Default;
             }
             int index = content.Length - 1;
-            int line = content.Count(c => c == NewLineCharacter); // Assume there is no new line at the end of the file, count(line) = count(newline) + 1
+            int line = Regex.Matches(content, Environment.NewLine).Count;
```

Comment:
using `Environment.NewLine` is not correct. Markdown content is not necessarily be `\r\n` under Windows.